src/components/Header/HeaderContact.js

```diff
@@ -33,7 +33,7 @@ const HeaderContact = ({
     }
     return (
       <GlobalButton testID="onPressIos" buttonStyle={styles.noPaddingLeft} onPress={onPress}>
-        <View testID="ios" style={styles.content(-8)}>
+        <View testID="ios" style={styles.content(-6)}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in the style parameter from -8 to -6 might affect the layout and alignment of the component. Please ensure that this change doesn't break the UI or make it inconsistent with other parts of the application.
```diff
- <View testID="ios" style={styles.content(-8)}>
+ <View testID="ios" style={styles.content(-6)}>
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->